src/components/add-movie/add-movie.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./add-movie.scss";
+
+const AddMovie = () => {
+    return (
+        <a href={"/"} className={"add-movie-link"}>add movie</a>
+    )
+}
```

Comment:
You can write simpler:
`const AddMovie = () => (<a href={"/"} className={"add-movie-link"}>add movie</a>);
`